### ISSUE TITLE
Bioconda package

### ISSUE THREAD

**Issue by muslih14** [GATB/simka#5]
I'd like to congratulate the developers on a great metagenomics software tool. I'd like to recommend the developers add simka to the bioconda channel as a package as it would enable easier installation and adoption by the community.

Best wishes,
Muslih.

**Comment by nick-youngblut** [comment]
There is a [simka bioconda recipe](https://bioconda.github.io/recipes/simka/README.html), which seems to have existed for a little while now, which makes me wonder whether this repo is still maintained...

There is an issue with the bioconda recipe, which stems from how `simka` is implemented. More specifically, if the user tries to run `simka` from the conda env (`simka` is in the PATH), then `simka` complains:

> EXCEPTION: Unable to get the real path for 'simka'

Why is the real path necessary?

**Comment by lecorguille** [comment]
Indeed, I get the same message if I use simka from the $PATH without absolute or related path.
It seem that simka need to know where it's to find some other alongside scripts, binaries or libraries?

If you use this:
```
$(which simka) -in example/simka_input.txt -out /tmp/results -out-tmp /tmp/temp_output
```
It works!

**Comment by nick-youngblut** [comment]
@lecorguille any progress on fixing this simka path issue? `$(which simka)` is a confusing hack that each user has to figure out.

**Comment by lecorguille** [comment]
No idea. It's a "bug" that have to be firgure by the sinka dev team.
I'm afraid that I'm enough confortable with c++ to tackle the issue myself.

They should implement either a way to add all their side binaries in the $PATH or prepend their side binaries calls with a sort of `realpath $(dirname $0)` (I :heart: bash)

**Comment by clemaitre** [comment]
Thank you @nick-youngblut and @lecorguille for bringing this issue back to the forefront. I had forgotten about it and this is indeed an important issue to make simka easier to use. Now that we have at last some human resources on simka, we will try to fix this soon.

Best,
Claire

**Comment by nick-youngblut** [comment]
Great! As an FYI on simka: I just used simka to identify a few outlier samples in my metagenome dataset. simka is especially helpful when most reads don't map to reference databases, so such ref-based methods may not reveal outliers.

**Comment by clemaitre** [comment]
This issue has been fixed in release 1.5.3 (commit 3bac22da85eea992f4ba6c6993740dee06aabc97)
